### PR TITLE
chore: merge main to dev

### PR DESCRIPTION
**Pull request by kuojianlu**

### CODE REVIEW

**Review by wenytang-ms — APPROVED** [review]

**Review by Alive-Fish — DISMISSED** [review]
Approve for CLI part.

File: packages/fx-core/tests/plugins/resource/function/unit/expectedBicepFiles/functionConfiguration.bicep [hunk 1]
```diff
@@ -0,0 +1,81 @@
+param functionAppName string
```
Comment (swatDong) [inline packages/fx-core/tests/plugins/resource/function/unit/expectedBicepFiles/functionConfiguration.bicep:1]: This `functionConfiguration.bicep` file was removed from dev branch, is it needed any more?

File: packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.all.bicep [hunk 2]
```diff
@@ -0,0 +1,45 @@
+param simpleAuthWebAppName string
```
Comment (swatDong) [inline packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.all.bicep:1]: This `simple_auth_configuration.all.bicep` file was removed from dev branch.

File: packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.only.bicep [hunk 3]
```diff
@@ -0,0 +1,42 @@
+param simpleAuthWebAppName string
```
Comment (swatDong) [inline packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.only.bicep:1]: This `simple_auth_configuration.only.bicep` file was removed from dev branch.

**Comment by swatDong** [comment]
Note: **DO NOT squash** when merging main into dev.

File: packages/fx-core/tests/plugins/resource/function/unit/expectedBicepFiles/functionConfiguration.bicep (hunk 1 above)
Comment (kuojianlu) [reply packages/fx-core/tests/plugins/resource/function/unit/expectedBicepFiles/functionConfiguration.bicep:1]: Good catch! These test files should be removed together.

Comment (kuojianlu) [reply packages/fx-core/tests/plugins/resource/function/unit/expectedBicepFiles/functionConfiguration.bicep]: Removed.

File: packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.all.bicep (hunk 2 above)
Comment (kuojianlu) [reply packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.all.bicep]: Removed.

File: packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.only.bicep (hunk 3 above)
Comment (kuojianlu) [reply packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.only.bicep]: Removed.

**Review by swatDong — APPROVED** [review]

**Review by wenytang-ms — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by KennethBWSong — APPROVED** [review]

**Pull request merged**